assets/tests/components/ladder.test.tsx

```diff
@@ -602,7 +602,9 @@ describe("ladder", () => {
       </StateDispatchProvider>
     )
 
-    await userEvent.click(result.getByText("N/A"))
+    await userEvent.click(
+      result.getByText("N/A", { selector: ".m-vehicle-icon__label" })
+    )
```

Comment:
**question:** why is adding the `selector` necessary? I can't quite figure out what changed that would cause problems with the old way that the element was selected.

Reply:
The operator's name is also "N/A". Now that it is wrapped in a `<span>` [here](https://github.com/mbta/skate/pull/1831/files#diff-1a657413769612430fbcd539c7d5cdd71b25ae44569ed9db7c235ad1b7c52812R178), it matches the `getByText` selector when it didn't before, which results in an error that there are multiple results for `result.getByText("N/A")` without the selector specified.